ambry-clustermap/src/test/java/com.github.ambry.clustermap/ClusterMapManagerTest.java

```diff
@@ -137,7 +141,7 @@ public void addNewPartition()
   }
 
   @Test
-  public void bestEffortAllocation()
+  public void bestEffortAllocationNonRackAware()
```

Comment:
the function name says bestEffort but you pass `false` for the `bestEfffort` arg. I'm a taf confused.